### PR TITLE
Feature/loginAndRegister

### LINKED ISSUE #1: Front - Login/Register
Create login/Register:

- [x] Add cookies support

- [x] Handle automatic redirection when people hit '/'

- [x] Check if cookies has valid JWT (need a new route for that).

- [x] Post registeration details ie username, password, registeration code to '/register'

- [x] Post login details to '/login' 

- [x] Save response token into cookies and user context. 

- [ ] Ask Justin how long the jwt is valid for.

- [x] After successful login/token get, route to home of respective user.

- [ ] Input validation

### LINKED ISSUE #2: Front - Industry Card and Card list
Component for Card and way to layout cards in list.
Rows and Columns of 3  per row.

- [ ] [example of the layout is here] (https://lawcareerstarter.com/lsnsw-virtual-fair-2020)

### PR DESCRIPTION
**Pull request by Mxchaeltrxn**
## Description

Create login and register card

### Dependencies added
Semantic ui css (required because react semantic ui is decoupled from semantic ui css).

Fixes #1 

## Browser Checklist

This PR has been tested in the following browsers:

- [ ] Chrome
- [ ] Firefox

### CODE REVIEW

File: website/src/style/_loginRegister.scss [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    // set image
+    background: #FAFAFA !important;
+    background-image: url(../Assets/loginMountain.jpg) !important;
+    /* Full height */
+    min-height: 100% !important; 
+  
+    /* Center and scale the image nicely */
+    background-position: center !important;
+    background-repeat: no-repeat !important;
+    background-size: cover !important;
```
Comment (Mxchaeltrxn) [inline website/src/style/_loginRegister.scss:23]: I'm unsure why you're using an `id` here and `!important`. These increase the specificity of the style and make it hard to override in the future. I also tested changing the id to a class and removing `!important` from lines 15, 16 and nothing seemed to change. 

Is this related to the css framework we're using?

File: website/src/pages/RegisterLogin/login.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    setInput((prevState) => ({ ...prevState, email }));
+  };
+
+  const handlePasswordUpdate = (event) => {
+    const password = event.target.value;
+    setInput((prevState) => ({ ...prevState, password }));
+  };
+
+  return (
+    <div className={props.className}>
```
[the remark this answers is not in the document]
Comment (metallicOxide) [reply website/src/pages/RegisterLogin/login.tsx:21]: Should be removed, was planning to make it a reusable component with different styling at first

File: website/src/pages/RegisterLogin/loginRegisterPage.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { Card } from "semantic-ui-react";
+import Login from "./login";
+import Register from "./register";
+
+function LoginRegisterPage() {
+  const [pageState, setPageState] = useState({
+    isLogin: true,
+    loginleft: 20,
+    registerleft: 550,
```
[the remark this answers is not in the document]
Comment (metallicOxide) [reply website/src/pages/RegisterLogin/loginRegisterPage.tsx:10]: the loginLeft and registerleft is done for the sliding animation. We could dynamically control classes on the states and have css classes to control their left and right

File: website/src/style/_loginRegister.scss (hunk 1 above)
Comment (metallicOxide) [reply website/src/style/_loginRegister.scss:23]: LoginRegisterPage isn't something we would want to reuse.

I have tried creating a <LoginRegisterPage> tag, however there was typescript errors.

I will remove the important tags if the css works without them... I was not working when I was working on the page

**Pull request merged**